src/components/create-from-builder/create-from-builder-info.html

```diff
@@ -16,6 +16,7 @@
           {{tag}}
         </span>
       </div>
+      <span>Version {{$ctrl.istag.annotations.version}}</span>
```

Comment:
Most image streams won't have a version annotation, so this will just show up as "Version " with no value in most cases.

Did you mean to add this to the create from builder form? It looks like the issue has a screenshot of a service class, not a builder image.